### ISSUE TITLE
VVT: de.schildbach.pte.exception.UnexpectedRedirectException: https://fahrplan.vvt.at/bin/mgate.exe -> https://smartride.vvt.at/bin/mgate.exe

### ISSUE THREAD

**Issue by whowantsmybigdata** [schildbach/public-transport-enabler#341]
### Query
- NetworkId: `VVT`
- From: `Location{STATION, 476424300, 47.2468990/11.4710270, name=Aldrans Aste, products=[BUS]}`
- Via: `null`
- To: `Location{STATION, 476423400, 47.2581350/11.4078150, name=Innsbruck Olympiaworld, products=[BUS]}`
- Date: `Wed Dec 02 16:21:12 GMT+01:00 2020`
- Departure: `true`
- Products: `[HIGH_SPEED_TRAIN, REGIONAL_TRAIN, SUBURBAN_TRAIN, SUBWAY, TRAM, BUS, FERRY, CABLECAR, ON_DEMAND]`
- Optimize for: `LEAST_DURATION`
- Walk Speed: `NORMAL`

### Error
```
de.schildbach.pte.exception.UnexpectedRedirectException: https://fahrplan.vvt.at/bin/mgate.exe -> https://smartride.vvt.at/bin/mgate.exe
de.schildbach.pte.util.HttpClient.getInputStream(HttpClient.java:288)
de.schildbach.pte.util.HttpClient.get(HttpClient.java:205)
de.schildbach.pte.AbstractHafasClientInterfaceProvider.jsonTripSearch(AbstractHafasClientInterfaceProvider.java:583)
```

### Additional information
[Please modify this part]

**Comment by schildbach** [comment]
Fixed via c082372c1951268c0844ce04ca8f7c4f0fc1f273.

**Comment by whowantsmybigdata** [comment]
Oh wow, that was fast (and easy)! 
Seems like I'm the only one in Tyrol using this app as my girlfriend just mentioned this change in the VVT-(API)-address appeared already months ago...
Anyway, thanks a lot!!
